tpgtools/override.go

```diff
@@ -25,6 +25,12 @@ import (
 
 type OverrideType string // enum
 
+// Product-level Overrides
+const (
+	// if the basepath is already generated by mm, skip it
+	ProductSkipBasePath OverrideType = "SKIP_BASEPATH_SINCE_ALREADY_PRESENT"
```

Comment:
I wouldn't include a reason in the override. `SKIP_BASEPATH` is sufficient.

We can include comments for the reason for each product, because this may not be the only reason we do it